test/double_spend_test.go

```diff
@@ -31,11 +31,6 @@ func TestDoubleSpend(t *testing.T) {
 	for _, tc := range tt {
 		t.Run(tc.name, func(t *testing.T) {
 			address, privateKey := getNewWalletAddress(t)
-
-			generate(t, 100)
```

Comment:
I think if we can remove this, then we can also remove lines 🤔 
- test/endpoint_test.go:63
- test/endpoint_test.go:203
- test/endpoint_test.go:405
- test/endpoint_test.go:451
- test/endpoint_test.go:638

Reply:
changed